examples/java/src/main/java/org/apache/ignite/example/table/RecordViewPojoExample.java

```diff
@@ -42,81 +39,73 @@ public static void main(String[] args) throws Exception {
-                    "CREATE TABLE accounts ("
-                            + "accountNumber INT PRIMARY KEY,"
-                            + "firstName     VARCHAR,"
-                            + "lastName      VARCHAR,"
-                            + "balance       DOUBLE)"
-            );
-        }
-
-        //--------------------------------------------------------------------------------------
-        //
-        // Creating a client to connect to the cluster.
-        //
-        //--------------------------------------------------------------------------------------
-
-        System.out.println("\nConnecting to server...");
-
         try (IgniteClient client = IgniteClient.builder()
                 .addresses("127.0.0.1:10800")
                 .build()
         ) {
-            //--------------------------------------------------------------------------------------
-            //
-            // Creating a record view for the 'accounts' table.
-            //
-            //--------------------------------------------------------------------------------------
-
-            RecordView<Account> accounts = client.tables()
-                    .table("accounts")
-                    .recordView(Account.class);
-
-            //--------------------------------------------------------------------------------------
-            //
-            // Performing the 'insert' operation.
-            //
-            //--------------------------------------------------------------------------------------
-
-            System.out.println("\nInserting a record into the 'accounts' table...");
-
-            Account newAccount = new Account(
-                    123456,
-                    "Val",
-                    "Kulichenko",
-                    100.00d
-            );
-
-            accounts.insert(null, newAccount);
-
-            //--------------------------------------------------------------------------------------
-            //
-            // Performing the 'get' operation.
-            //
-            //--------------------------------------------------------------------------------------
-
-            System.out.println("\nRetrieving a record using RecordView API...");
-
-            Account account = accounts.get(null, new Account(123456));
-
-            System.out.println(
-                    "\nRetrieved record:\n"
-                        + "    Account Number: " + account.accountNumber + '\n'
-                        + "    Owner: " + account.firstName + " " + account.lastName + '\n'
-                        + "    Balance: $" + account.balance);
-        } finally {
-            System.out.println("\nDropping the table...");
-
-            try (
-                    Connection conn = DriverManager.getConnection("jdbc:ignite:thin://127.0.0.1:10800/");
-                    Statement stmt = conn.createStatement()
-            ) {
-                stmt.executeUpdate("DROP TABLE accounts");
+            try {
```

Comment:
Please do not change unrelated examples. This example should be reverted. If you want to overcome the broken compatibility issue (https://issues.apache.org/jira/browse/IGNITE-27280) then please use the latest Ignite 3 build from `main`.